docs/release.md

```diff
@@ -22,6 +22,19 @@ git push origin $REL_VER
 
 Or if you want to build/push images for local development.
 
+**1.  Prepare JVS image**
+
+Pre-built JVS release images can be found from `us-docker.pkg.dev/abcxyz-artifacts/docker-images`. Please ensure to select the
+image compatible with your JIRA plugin.
+
+If you want to build a JVS image for local development, follow [this guidance](https://github.com/abcxyz/jvs/blob/main/docs/release.md#manually-release-images).
+
+**2.  Overlay JIRA plugin image**
+
+Update [Dockerfile](https://github.com/abcxyz/jvs-plugin-jira/blob/main/Dockerfile)
```

Comment:
Do we want this in the "New Release" section as well?

Reply:
Yes, if base JVS image need to be changed. I will merge this PR first and will send out a followed up PR to update the New Release section

Reply:
Could you fix this at the same time? https://github.com/abcxyz/jvs-plugin-jira/issues/32 Thank you!